pkg/e2e/verify/namespace_webhook.go

```diff
@@ -157,6 +186,80 @@ var _ = ginkgo.Describe(namespaceWebhookTestName, func() {
 				Expect(err).NotTo(HaveOccurred())
 			}
 		}, viper.GetFloat64(config.Tests.PollingTimeout))
+
+		util.GinkgoIt("dedicated admin can not create PrometheusRule", func(ctx context.Context) {
+			for privilegedNamespace := range PRIVILEGED_NAMESPACES {
+				prome2etestrule := e2eCreatePrometheusRule(prometheusName, privilegedNamespace, DUMMY_USER, "dedicated-admins", h)
+				err := wait.PollImmediate(5*time.Second, 1*time.Minute, func() (bool, error) {
+					_, err := h.Prometheusop().MonitoringV1().PrometheusRules(privilegedNamespace).Create(ctx, prome2etestrule, metav1.CreateOptions{})
+					if err != nil && errors.IsForbidden(err) {
+						log.Printf("failed creating prometheus rules: %v", err)
```

Comment:
This log could end up being misleading if someone's reviewing raw test logs. The "Failure" in this case is good and indicative of test success. I'd lean towards removing this log.